### PR TITLE
Mention throughput as a potential drawback of mDNS failure

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -256,8 +256,8 @@ resolved if the remote endpoint is too many segments away.
 
 When mDNS fails, ICE will attempt to fall back to either NAT hairpin,
 if supported, or TURN relay, if not. As noted in {{IPHandling}}, this may
-result in increased media latency and reduced connectivity/increased cost
-(depending on whether the application chooses to use TURN).
+result in increased media latency, reduced connectivity, reduced throughput
+and increased cost (depending on whether the application chooses to use TURN).
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: or?

Comment (juberti) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: I think this needs to be something like  "increased media latency, increased cost, and reduced throughput (if the application chooses to use TURN), or reduced connectivity (if not)."

Comment (youennf) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: I was thinking that, in theory, hairpinning could also increase latency and reduce throughput.
Agreed that reduced connectivity should probably be separated.
I'll update to make things more precise.

Comment (lgrahl) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: > I was thinking that, in theory, hairpinning could also increase latency and reduce throughput.

Not only in theory. This happens a lot in cable networks due to DS-Lite Gateways.

Comment (juberti) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: This seems kind of wordy now. I don't think we need to be super precise, we can just list the things as potential problems without getting into all the minutiae.

Happy to take a shot at it if useful.

Comment (youennf) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: New shorter version

**Pull request merged**